playbooks/provisioner/openstack/cleanup.yaml

```diff
@@ -0,0 +1,6 @@
+---
+- name: Cleanup OpenStack resources
+  hosts: localhost
+  roles:
+      - openstack/cleanup/compute
```

Comment:
I would much more prefer to stick to OpenStack naming (neutron, nova, glance, cinder)...